### PR TITLE
PEP8 legacy fixes

### PR DESCRIPTION
**Pull request by aioue**
##### SUMMARY

PEP8 legacy fixes
ec2_ami
ec2_elb_facts
ec2_eni_facts
ec2_snapshot
ec2_tag

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

ec2_ami
ec2_elb_facts
ec2_eni_facts
ec2_snapshot
ec2_tag

##### ANSIBLE VERSION

```
ansible 2.4.0 (pep8-legacy 6f923e67c7) last updated 2017/07/07 17:18:34 (GMT +100)
  config file = None
  configured module search path = [u'/Users/tom/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/tom/repos/github/ansible/lib/ansible
  executable location = /Users/tom/repos/github/ansible/bin/ansible
  python version = 2.7.13 (default, Apr  4 2017, 08:47:57) [GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.38)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by s-hertel — APPROVED** [review]

File: lib/ansible/modules/cloud/amazon/ec2_ami.py [hunk 1]
```diff
@@ -476,7 +475,7 @@ def create_image(module, ec2):
             elif img.state == 'failed':
                 module.fail_json(msg="AMI creation failed, please see the AWS console for more details")
         except boto.exception.EC2ResponseError as e:
-            if ('InvalidAMIID.NotFound' not in e.error_code and 'InvalidAMIID.Unavailable' not in e.error_code) and wait and i == wait_timeout - 1:
+            if (('InvalidAMIID.NotFound' not in e.error_code and 'InvalidAMIID.Unavailable' not in e.error_code) and wait and i == wait_timeout - 1):
```
Comment (ryansb) [inline lib/ansible/modules/cloud/amazon/ec2_ami.py]: These added parens aren't needed.

**Review by ryansb — CHANGES_REQUESTED** [review]
One last tweak, then LGTM.

**Comment by willthames** [comment]
https://github.com/ansible/ansible/blob/devel/CODING_GUIDELINES.md#pep8-and-basic-style-checks says that we don't really do PRs just for PEP8

The reason I point this out is that, unless someone has reviewed https://ansible.sivel.net/pr/byfile.html for PRs that might clash, this could cause merge conflicts with PRs that are otherwise good to merge, increasing an already frustrating merge cycle.

**Comment by aioue** [comment]
Understood. Closing.

**Pull request closed without merging**